_unittest/test_21_Circuit.py

```diff
@@ -985,3 +985,10 @@ def test_51_import_asc(self):
         self.aedtapp.insert_design("ASC")
         asc_file = os.path.join(local_path, "example_models", test_subfolder, "butter.asc")
         assert self.aedtapp.create_schematic_from_asc_file(asc_file)
+
+    def test_52_create_page(self):
+        self.aedtapp.create_page()
+        assert self.aedtapp.GetNumPages() == 2
```

Comment:
@DaveTwyman you are not testing your method here. Your method is in the schematic. Why are you testing GetNumPages()? Also remember to assert the failure with assert not.

Reply:
Changed as requested with more testing of name types added